src/Exceptions/SubscriptionUpdateFailure.php

```diff
@@ -19,4 +32,31 @@ public static function incompleteSubscription(Subscription $subscription)
             "The subscription \"{$subscription->stripe_id}\" cannot be updated because its payment is incomplete."
         );
     }
+
+    /**
+     * Create a new SubscriptionUpdateFailure instance.
```

Comment:
I think this is a duplicate of of line 11

Reply:
This is correct.